Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/remesh.h

```diff
@@ -332,12 +334,27 @@ void isotropic_remeshing(const FaceRange& faces
 
 #ifdef CGAL_PMP_REMESHING_VERBOSE
   t.stop();
-  std::cout << "Remeshing done (size = " << target_edge_length;
-  std::cout << ", #iter = " << nb_iterations;
+  std::cout << "Remeshing done (#iter = " << nb_iterations;
   std::cout << ", " << t.time() << " sec )." << std::endl;
 #endif
 }
 
+// Overload when using target_edge_length for sizing
+template<typename PolygonMesh
+       , typename FaceRange
+       , typename NamedParameters = parameters::Default_named_parameters>
+void isotropic_remeshing(const FaceRange& faces
+                       , const double target_edge_length
+                       , PolygonMesh& pmesh
+                       , const NamedParameters& np = parameters::default_values())
+{
+  Uniform_sizing_field<PolygonMesh> sizing(target_edge_length, pmesh);
+  if (target_edge_length > 0)
+    isotropic_remeshing(faces, sizing, pmesh, np);
+  else
+    isotropic_remeshing(faces, sizing, pmesh, np.do_split(false).do_collapse(false));
```

Comment:
What is the behavior with `target_edge_length<=0`?

Reply:
Should be the same as the current behavior in master: https://github.com/CGAL/cgal/blob/807543e419012b491489f1b4cd4d7cf6083a1a2b/Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/remesh.h#L316

Only flips and smoothing.